### DISCUSSION TITLE
need help disabling eslint from giving these annoying errors "file is a commonJS module; it may be converted to an ES module"

### DISCUSSION THREAD

**Discussion by notfatdan** [LazyVim/LazyVim#3730] [Q&A]
![image](https://github.com/LazyVim/LazyVim/assets/84719438/793f8689-5975-47ab-84d5-627d92f32c3e)

return {
  "neovim/nvim-lspconfig",
  ---@class pluginlspopts
  opts = {
    ---@type lspconfig.options
     preferences = {
          disablesuggestions = true,
        },
    },
}

this is what ive tried, it doesnt work

**Comment by rubiin** [comment]
@lagiath try out the solutions from here https://www.reddit.com/r/neovim/comments/nv3qh8/how_to_ignore_tsserver_error_file_is_a_commonjs/

**Reply by notfatdan** [reply]
![image](https://github.com/LazyVim/LazyVim/assets/84719438/25b28b89-5fef-417f-ac5d-f6771ba0601c)

thanks that fixed the inline error but now i get spammed with these every time i type a character

**Reply by notfatdan** [reply]
i fixed it by uninstalling the eslint-lsp. thank god i can use my text editor again
